alpine-base-s6/Dockerfile.tmpl

```
@@ -20,7 +20,7 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.description="This service provides a base Alpine Linux container." \
       org.label-schema.architecture="x86_64" \
       org.label-schema.distribution="Alpine Linux" \
-      org.label-schema.distribution-version=$VERSION \
+      org.label-schema.distribution-version=%%UPSTREAM%% \
```

This is set as build var in the top level Makefile.conf
```suggestion
      org.label-schema.distribution-version=$UPSTREAM \
```